lk2nd/device/dts/msm8916/msm8916-qrd-9.dts

```diff
@@ -1,11 +1,22 @@
 // SPDX-License-Identifier: GPL-2.0-only
 
+#include <dt-bindings/gpio.h>
+#include <dt-bindings/dev/keys.h>
 #include <skeleton64.dtsi>
 #include <lk2nd.dtsi>
 
 / {
 	qcom,msm-id = <QCOM_ID_MSM8916 0>;
 	qcom,board-id = <QCOM_BOARD_ID_QRD 9>;
+
+	tlmm: tlmm {
```

Comment:
can you move this one to lk2nd.dtsi as well? I'd also move the additional includes there I guess to keep device files small'n'simple